io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,108 @@
+      "--socket=wayland",
+      "--env=QML_IMPORT_PATH=/app/qml",
+      "--socket=pulseaudio",
+      "--filesystem=host:ro",
+      "--talk-name=org.kde.klipper",
+      "--talk-name=org.kde.*"
+   ],
+   "cleanup-commands": [
+      "/app/cleanup-BaseApp.sh"
+   ],
+   "modules": [
+      {
+         "buildsystem": "cmake-ninja",
+         "name": "kiview",
+         "sources": [
+            {
+               "type": "git",
+               "url": "https://github.com/Nyre221/Kiview.git",
+               "commit": "a97244a4fdfccfb37755558291f1262313756a18"
+            }
+         ]
+      },
+      {
+         "name": "unrar",
+         "no-autogen": true,
+         "build-options": {
+            "strip": true
+         },
+         "make-install-args": [
+            "DESTDIR=$(FLATPAK_DEST)"
+         ],
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://www.rarlab.com/rar/unrarsrc-6.2.12.tar.gz",
+               "sha256": "a008b5f949bca9bb4ffa1bebbfc8b3c14b89df10a10354809b845232d5f582e5",
+               "x-checker-data": {
+                  "type": "anitya",
+                  "project-id": 13306,
+                  "url-template": "https://www.rarlab.com/rar/unrarsrc-$version.tar.gz"
+               }
+            }
+         ]
+      },
+      {
+         "name": "avahi",
+         "config-opts": [
+            "--disable-libdaemon",
+            "--enable-compat-libdns_sd",
+            "--disable-python",
+            "--disable-doxygen-doc",
+            "--with-distro=none",
+            "--disable-glib",
+            "--disable-gobject",
+            "--disable-introspection",
+            "--disable-gtk",
+            "--disable-gtk3",
+            "--disable-qt3",
+            "--disable-qt4",
+            "--disable-qt5",
+            "--disable-mono",
+            "--disable-monodoc",
+            "--disable-manpages",
+            "--disable-xmltoman",
+            "--disable-utils"
+         ],
+         "cleanup": [
+            "/bin"
+         ],
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://github.com/avahi/avahi/releases/download/v0.8/avahi-0.8.tar.gz",
+               "sha256": "060309d7a333d38d951bc27598c677af1796934dbd98e1024e7ad8de798fedda"
+            }
+         ]
+      },
+      {
+         "name": "libreoffice",
+         "buildsystem": "simple",
+         "build-commands": [
+            "chmod +x ./LibreOffice*AppImage",
+            "./LibreOffice*AppImage --appimage-extract",
+            "mv ./squashfs-root /app/bin/libreoffice-dir",
+            "ln -s /app/bin/libreoffice-dir/AppRun /app/bin/libreoffice"
+         ],
+         "sources": [
+            {
+               "type": "file",
+               "url": "https://appimages.libreitalia.org/LibreOffice-24.2.0.3.basic-x86_64.AppImage",
+               "sha256": "c8ca641329bcfff31dc16f0dc945728afeb6feb0425bb4096bcac2759413afc4"
```

Comment:
Add a `"dest-filename": "LibreOffice.AppImage"` here, then drop the `*` from the build commands above.

Reply:
Also a `"only-arches": ["x86_64"]` below